mysql-async/src/test/java/com/github/jasync/sql/db/mysql/StoredProceduresSpec.kt

```diff
@@ -96,7 +96,7 @@ class StoredProceduresSpec : ConnectionHelper() {
             ).get()
             val rows = connection.sendQuery(
                 """
-                  SELECT routine_name FROM INFORMATION_SCHEMA.ROUTINES WHERE routine_name="remTest"
+                  SELECT routine_name as routine_name FROM INFORMATION_SCHEMA.ROUTINES WHERE routine_name="remTest"
```

Comment:
Workaround for a breaking change in MySQL 8.0. [Reference](https://bugs.mysql.com/bug.php?id=84456)